CHANGELOG.md

```diff
@@ -2,6 +2,57 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+## [2.31.0](https://github.com/serverless/serverless/compare/v2.30.3...v2.31.0) (2021-03-23)
+
+### Features
+
+- **AWS API Gateway:** Remove deprecation for old naming convention ([#9160](https://github.com/serverless/serverless/pull/9160)) ([b530d6a](https://github.com/serverless/serverless/commit/b530d6a288a9881c12382c8004447b94a80f1847)) ([Piotr Grzesik](https://github.com/pgrzesik))
+- **Variables:**
+  - Disallow `provider.variableSyntax` with new resolver ([#9140](https://github.com/serverless/serverless/pull/9140)) ([5a2da44](https://github.com/serverless/serverless/commit/5a2da444ea5f896f56cf03e509631d2df548457c)) ([Mariusz Nowak](https://github.com/medikoo))
+  - Meaningfully reject not resolvable `provider.name` ([#9140](https://github.com/serverless/serverless/pull/9140)) ([f67d95a](https://github.com/serverless/serverless/commit/f67d95a553be8986b8b21c164f9d292f4fca21cd)) ([Mariusz Nowak](https://github.com/medikoo))
+
+### Bug Fixes
+
+- **Packaging:** Ensure to properly exclude dependencies when NODE_ENV set ([#9142](https://github.com/serverless/serverless/pull/9142)) ([6e3e21c](https://github.com/serverless/serverless/commit/6e3e21cafaa053a0f30239513eac5c7a725b7694)) ([Piotr Grzesik](https://github.com/pgrzesik))
+- **CLI:**
+  - Recognize "--stage" as provider agnostic option ([#9151](https://github.com/serverless/serverless/pull/9151)) ([271ac82](https://github.com/serverless/serverless/commit/271ac8281c10e23ce609165582b3b996b60d5bd1)) ([Mariusz Nowak](https://github.com/medikoo))
+  - Do not show options order deprecation info with help request ([#9151](https://github.com/serverless/serverless/pull/9151)) ([ad8f9b0](https://github.com/serverless/serverless/commit/ad8f9b059740bb2581445457b6751e918a57e585)) ([Mariusz Nowak](https://github.com/medikoo))
+  - Ensure all expected command schemas in collections ([#9168](https://github.com/serverless/serverless/pull/9168)) ([031ca4d](https://github.com/serverless/serverless/commit/031ca4dd2aa35b3a15e00db0601a11b7b25d3b4d)) ([Mariusz Nowak](https://github.com/medikoo))
```

Comment:
Let's remove that one (it was fix against `master` and not against previous release)